### PR TITLE
Digital fund prototype

### PR DESCRIPTION
**Pull request by davidrapson**

### CODE REVIEW

File: controllers/apply/digital-funding-demo/form-model.js [hunk 1]
```diff
@@ -106,12 +106,11 @@ module.exports = strandNumber => {
         title: `Digital Funding Strand ${strandNumber} (Demo)`,
         pageAccent: 'blue',
         heroSlug: 'whizz-kidz',
-        hidePagination: true,
         shortCode: `DF${strandNumber}-ALPHA`,
         steps: strandSteps[`strand${strandNumber}`],
         processor: processor,
         startPage: {
-            template: path.resolve(__dirname, './startpage')
+            path: `/funding/programmes/digital-funding-demo/strand-${strandNumber}`
```
Comment (davidrapson) [inline controllers/apply/digital-funding-demo/form-model.js]: As this is the one part of "real" code we're touching, could we name this more explicitly, maybe `urlPath`. The `template` property is a path too, albeit a file path, but still keen to make the distinction clear.

File: assets/js/modules/forms.js [hunk 2]
```diff
@@ -88,9 +88,21 @@ function initApplicationForms() {
...
+            e.preventDefault();
+            mockFormEl.classList.toggle('is-submitted');
+        });
+    }
+}
+
 function init() {
     initLegacyForms();
     initApplicationForms();
+    initMockEmailSignup();
```
Comment (davidrapson) [inline assets/js/modules/forms.js]: Given how throwaway this is could we pop this in an inline `<script>` on the page?

File: assets/sass/components/_forms.scss [hunk 3]
```diff
@@ -482,3 +494,43 @@
...
+.email-signup {
+    background-color: palette('pale-grey');
+    padding: $spacingUnit;
+
+    input[type='submit'] {
+        display: inline-block;
+        vertical-align: middle;
+    }
+
+    .post-submit {
```
Comment (davidrapson) [inline assets/sass/components/_forms.scss]: :neckbeard: Mixing BEM classes makes me twitch a little…but this is fine for now.

File: controllers/apply/digital-funding-demo/form-model.js (hunk 1 above)
Comment (mattandrews) [reply controllers/apply/digital-funding-demo/form-model.js]: Good call – done.

**Pull request merged**